docs/data-ai/ai/alert.md

```diff
@@ -64,83 +76,97 @@ For example, if using the YOLOv8 model (named `yolo`) for hardhat detection, you
+1. Now you're ready to enable data capture on your detector camera.
    Locate the `objectfilter-cam` panel.
 
-3. Click **Add method**.
+1. Click **Add method**.
    Click the **Type** dropdown and select **ReadImage**.
    Set the capture frequency to `0.2` images per second (equivalent to one image every 5 seconds).
    You can always change the frequency to suit your use case.
    Set the **MIME type** to `image/jpeg`.
 
 ## Set up alerts
 
-[Triggers](/data-ai/data/advanced/alert-data/) allow you to send webhook requests or email notifications when certain events happen.
-
-You can use the **Data has been synced to the cloud** (`part_data_ingested`) trigger to send alerts whenever an image with an anomaly detection is synced to the cloud from your object filter camera.
+Triggers send webhook requests or email notifications when certain events happen.
 
-Set up the trigger with a webhook or with Viam's built-in email alerts which sends a generic email letting you know that data has been synced.
+You can use the **Data has been synced to the cloud** (`part_data_ingested`) trigger type to send alerts whenever an image syncs to the cloud from your filtered camera.
+Because the filter only syncs images that contain an anomaly, this trigger sends an alert when an anomaly occurs.
 
 ### Configure a trigger on your machine
 
-Now it's time to configure a trigger so that you get an email when a person is not wearing a hard hat.
+Follow these steps to configure a trigger to alert when `filtered-camera` syncs an image:
+
+1. Go to the **CONFIGURE** tab of your machine on the [Viam app](https://app.viam.com).
+   Click the **+** (Create) button in the left side menu and select **Trigger**.
 
-Go to the **CONFIGURE** tab of your machine on the [Viam app](https://app.viam.com).
-Click the **+** (Create) button in the left side menu and select **Trigger**.
+1. Enter a name and click **Create**.
 
-Name the trigger and click **Create**.
+1. In the **Type** dropdown, select **Data has been synced to the cloud**.
 
-Select trigger **Type** as **Data has been synced to the cloud** and **Data Types** as **Binary (image)**.
+1. In the **Data Types** dropdown, select **Binary (image)**.
 
-{{<imgproc src="/tutorials/helmet/trigger.png" resize="x300" declaredimensions=true alt="The trigger created with data has been synced to the cloud as the type and binary (image) as the data type." class="shadow" >}}
+   {{<imgproc src="/tutorials/helmet/trigger.png" resize="x300" declaredimensions=true alt="The trigger created with data has been synced to the cloud as the type and binary (image) as the data type." class="shadow" >}}
 
-To configure notifications, either
+1. To add a notification method, add an entry to the **Webhooks** or **Email** sub-panels:
 
-- add a webhook and enter the URL of your custom cloud function
-- add an email address to use Viam's built-in email notifications
+   - To add an email notification:
```

Comment:
This level of nesting looks odd. Remove these bullets and just use a new paragraph to differentiate.